buildout.cfg

```diff
@@ -1,33 +1,8 @@
 [buildout]
-extends =
-    https://raw.github.com/collective/buildout.plonetest/master/test-4.x.cfg
-    https://raw.github.com/collective/buildout.plonetest/master/qa.cfg
-
+extends = https://raw.githubusercontent.com/collective/buildout.plonetest/master/test-5.x.cfg
```

Comment:
I wondered about the changes in theTravis file too, but the 4.3 tests pass: https://travis-ci.org/collective/collective.contentrules.mailtogroup/jobs/120451498